### ISSUE TITLE
Chrome 58+ requires filling SubjectAltName

### ISSUE THREAD

**Issue by jkubos** [latentflip/dev-cert-authority#5]
It seems that since Chrome 58 and Firefox 48 it is required to have set field SubjectAltName in certificates. See https://textslashplain.com/2017/03/10/chrome-deprecates-subject-cn-matching/ for details.

It seems that solution is to add some parameters to openssl: https://security.stackexchange.com/questions/74345/provide-subjectaltname-to-openssl-directly-on-command-line

Can you take a look? Thank you!

**Comment by apkrasny** [comment]
I fixed this in [my fork](https://github.com/apkrasny/dev-cert-authority/commit/2de162a9bec37f744fa653b4c62c470a8c84fbed) since this repo seems inactive.

**Comment by xdumaine** [comment]
@apkrasny would you open a PR for that? @latentflip could you help get that merged and published?

**Comment by complyue** [comment]
ping! it's really annoying that I re-discover this after one and a half years when my cert expires. this should really be in the official repo.

**Comment by latentflip** [comment]
@complyue sounds like you've had a year and half of great service from this freely provided open source project 😄 .

I guess I missed this the first time around, if anyone wants to open a PR I can merge it in.

**Comment by complyue** [comment]
@latentflip I'm really grateful. As with other best things in life, like free air, you almost always forget its existence when it keeps benefitting you.

I just recall that I actually used @danwashusen 's fork years back, for it adds the main domain with wild sub-domains at the same time.

@danwashusen, I'd like your fork to be merged back into @latentflip 's more official repo, if you see this please give your opinion, or in case you missed this discussion, I'd like to draft a PR based on your fork and I think that's how open source is supposed to work.

Cheers you great guys!

**Comment by danwashusen** [comment]
Yep, my fork is working and we still use it... Unfortunately I updated the readme in my commit so its not really in a state that can be merged but should be very easy to port...

https://github.com/apkrasny/dev-cert-authority/compare/master...danwashusen:master

**Comment by complyue** [comment]
@danwashusen I think it's a "go" from you, and submitted PR #6 , which excludes modification to the installation instruction.

**Comment by complyue** [comment]
ping! @latentflip please review PR #6 and see if you can accept it?
